### DISCUSSION TITLE
Не генерируются результаты

### DISCUSSION THREAD

**Discussion by Tamika226** [nonealexq/jmeter-allure-reporting#12] [Q&A] [answered]
Привет!
Стянула себе проект попробовать, при попытке запуска выдает вот такое 
Попробовала настроить на своем проекте - то же самое. В чем может быть причина?
Пути к файлу со скриптом и к папке результатов задала корректно

![image](https://github.com/nonealexq/jmeter-allure-reporting/assets/77970707/c266c8b9-5229-4694-a109-2510decd9278)

**Comment by nonealexq** [comment, marked as answer]
Привет! Вот это делали?
https://github.com/nonealexq/jmeter-allure-reporting#quick-start-via-gui-mode

Сделаейте скрин своего User Defined Variables
